mmv1/products/compute/TargetHttpsProxy.yaml

```diff
@@ -115,7 +124,6 @@ properties:
       resource: 'SslCertificate'
       imports: 'selfLink'
       description: 'The SSL certificate URL or Certificate Manager certificate resource URL used by this TargetHttpsProxy'
-    custom_expand: 'templates/terraform/custom_expand/array_resourceref_with_validation.go.erb'
```

Comment:
We can't quite revert to default handling here for backwards compatibility reasons (which has triggered the conversion test to fail), but we can add handling the patterns of the new resource type that would have been rejected previously by copying https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/custom_expand/array_resourceref_with_validation.go.erb and inserting them between L23/L24. Roughly along the lines of:

```
regPat, _ := regexp.Compile("//certificatemanager.googleapis.com/projects/(.*))/locations/(.*))/certificates/(.*)") 
if regPat.MatchString(raw.(string)) {
  req = append(req, raw.(string))
  continue 
}
// original tpgresource.ParseGlobalFieldValue call
```

That will preserve the current behaviour where the shortname is treated as an sslCertificate, and specific forms are recognised as their respective resources.